samples/batch/transcription-enabled-storage/Connector/Serializable/TextAnalytics/TextAnalyticsSentence.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="TextAnalyticsSentence.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector.Serializable.TextAnalytics
+{
+    public class TextAnalyticsSentence
```

Comment:
This class contains only one member, is it really useful to have it as a separate class?

Reply:
Same as above: is used for deserialization/serialization.